search-app/src/cmr/search/services/acls/collections_cache.clj

```diff
@@ -16,16 +23,39 @@
 (defn create-cache
   "Creates a new empty collections cache."
   []
-  (mem-cache/create-in-memory-cache))
+  ;;(mem-cache/create-in-memory-cache)
+  (redis-cache/create-redis-cache {:keys-to-track [:collections-for-gran-acls] :ttl (* 15 60)}))
 
-(def cache-key
-  :collections-for-gran-acls)
+(defn- clj-times->time-strs
```

Comment:
if this is meant to be a reusable func, can you give example of its use here? Or if there is a unit test of this, that would work too

Reply:
I have since written a test for it, it takes date objects (clojure time objects) and turns them into strings so they can be saved in redis. The other function reverse the action.